alerts/alerts.go

```diff
@@ -99,7 +105,7 @@ func (s *Severity) UnmarshalJSON(b []byte) error {
 }
 
 // RegisterAlert implements the Alerter interface.
-func (m *Manager) RegisterAlert(_ context.Context, alert Alert) error {
+func (m *Manager) RegisterAlert(ctx context.Context, alert Alert) error {
 	if alert.ID == (types.Hash256{}) {
```

Comment:
I think you added a validator right? We can reuse it here

Reply:
All the validation is now in `RegisterAlert`.